dist/READMEpreview.md

```diff
@@ -0,0 +1,157 @@
+# OVN dev preview on OpenShift/origin/kubernetes
+
+You can preview ovn overlay networks on an OpenShift 3.11 cluster by replacing
+the installed SDN: Multitenant software with openvswitch/ovn components.
+
+NOTES:
+- This is Development Preview, not Production, and it will change over time.
+- There is no upgrade path. New install only.
+- This is limited to a single cluster master
+- This is limited to a single cluster network cidr range.
+- This is limited to OKD-3.11 (OKD-4.0 will build, install and work differently)
+
+
+## Installation:
+
+Install OKD-3.11 as instructed in the OKD documentation. Make the following change
+before running the cluster install playbook:
+1. In the ansible host file for the cluster change:
+```
+os_sdn_network_plugin_name='redhat/openshift-ovs-multitenant'
+```
+  to
+```
+os_sdn_network_plugin_name='cni'
+```
+
+When the cluster install is complete and the cluster is up there will
+be no cluster networking. The openshift-sdn and openshift-ovs daemonsets
+are, however, installed and must be removed.
+
+```
+$ ssh root@master.node.com   <<<----- the cluster master node
+# oc project openshift-sdn
+# oc delete ds openshift-sdn
```

Comment:
should be `oc delete ds sdn oc delete ds ovs`

Reply:
@zhaozhanqi will change